.travis.yml

```diff
@@ -18,22 +18,26 @@ addons:
 install:
   - ""
 
+# JDKs used in builds must be supported by Tycho. Check https://wiki.eclipse.org/Tycho/Release_Notes to find supported versions.
 matrix:
   fast_finish: true
   include:
+    # JDK 8 (oldest supported Java version of eclipse-cs)
```

Comment:
`JDK 8 (minimal supported Java version of eclipse-cs)`

Reply:
Done.